src/Configuration.ts

```diff
@@ -106,6 +108,7 @@ const defaultConfiguration: Configuration = {
 
   enableAnchoring: false,
   anchorIntervalInSeconds: 30,
+  purgeTransactionsInSeconds: 600,
```

Comment:
I don't think any value will really work well. I think its a flaw in that by default our settings mean that the reader needs to catch up to the blockchain from some set minimumBlockHeight. If the node is caught up it shouldn't be an issue or if the minimumBlockHeight is set relatively close to the current block to begin with.

Reply:
Maybe there should be a way for the reader to signal that it's up to date?

Reply:
```
message: requestPurgeStaleTransactions -> BlockReader:verifyUpToDate -> publish purgeStaleTransactions
```

Reply:
Actually. Looking at the issue again, the issue describes a different solution than the RDD doc I think. The issue describes a solution using dates, the RDD describes a solution using blockHeights. The blockHeights solution would not have this issue as it would wait for the reader to catch up and pass the txId createBlockHeight before wiping the transaction.

Reply:
Issue update: https://github.com/poetapp/node/issues/707#issuecomment-439519734